File: day09/day09.js
import { rawInput } from './input.js';
// import { exampleInput as rawInput } from './input.js';
import IntCode from './IntCode.js';

const program = rawInput.split(',').map(Number);

const { output: partOne } = new IntCode('A', program, 1);
const { output: partTwo } = new IntCode('A', program, 2);

document.getElementById('partOne').appendChild(document.createTextNode(partOne));
document.getElementById('partTwo').appendChild(document.createTextNode(partTwo));

File: day09/day09.py
from intcode import Intcode


def parse_input():
    with open("input.txt") as f:
        return [int(n) for n in f.read().split(",")]


program = parse_input()


senserProgram = Intcode(program)
senserProgram.run(1)

senserProgram = Intcode(program)
senserProgram.run(2)
